examples/embedded-nextjs/src/steps/Connect.tsx

```diff
@@ -0,0 +1,121 @@
+  const [isLoading, setIsLoading] = useState(false);
+  const [isShownAdvancedOptions, setIsShownAdvancedOptions] = useState(false);
+
+  useEffect(() => {
+    initSpireKey({
+      hostUrl: wallet,
+      useRAccount: true,
+    });
+  }, [wallet]);
+  const onConnect = async () => {
+    try {
+      setIsLoading(true);
+      const account = await connect(networkId, chainId as ChainId);
+      setAccount(account);
+      setAccount(await account.isReady());
+    } catch (e) {
+      console.warn('User canceled', e);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+  return (
+    <>
+      <CardContainer hasPadding>
+        <CardContentBlock
+          visual={<ProductIcon.QuickStart size="xl" />}
+          title="Step 1: Connect your account"
+          description={<ExampleStepper step={0} />}
+        >
+          <Stack flexDirection="column" gap="md">
+            <Select
+              label="Chain"
+              onSelectionChange={(c) => setChainId(c as ChainId)}
+              selectedKey={chainId}
+            >
+              {Array(20)
+                .fill(1)
+                .map((_, i) => (
+                  <SelectItem key={i}>{i.toString()}</SelectItem>
+                ))}
+            </Select>
+            <Button
+              variant="outlined"
+              onPress={() => setIsShownAdvancedOptions(!isShownAdvancedOptions)}
+              className={stackedButtonClass}
+            >
+              {isShownAdvancedOptions
+                ? 'Hide Advanced Options'
+                : 'Show Advanced Options'}
+            </Button>
+          </Stack>
+        </CardContentBlock>
+        {isShownAdvancedOptions ? (
+          <CardContentBlock
+            isNewSection
+            titleAs="h5"
+            title="Advanced Options"
+            description="These options allow you to test this dApp using your local SpireKey instance. Additionally you can opt to use a different network."
+          >
+            <Stack flexDirection="column" gap="md">
+              <Select
+                label="Wallet"
+                onSelectionChange={(w) => setWallet(w as string)}
+                selectedKey={wallet}
+              >
+                <SelectItem key="https://spirekey.kadena.io/">
+                  SpireKey
+                </SelectItem>
+                <SelectItem key="http://localhost:1337/">Local</SelectItem>
+              </Select>
+              <Select
+                label="Network"
+                onSelectionChange={(n) => setNetworkId(n as string)}
+                selectedKey={networkId}
+              >
+                <SelectItem key="mainnet01">Mainnet</SelectItem>
+                <SelectItem key="testnet04">Testnet</SelectItem>
+                <SelectItem key="development">Devnet</SelectItem>
+              </Select>
+            </Stack>
+          </CardContentBlock>
+        ) : null}
+      </CardContainer>
+      <CardFooter>
```

Comment:
This should probably be in the CardContainer

Reply:
No it actually was meant like this, the primary buttons are supposed to be outside of the card (checked with Isa)